### PR TITLE
[Project Submission] NFT Import

### PR DESCRIPTION
**Pull request by zapaz**
NFT import is a WordPress Plugin allowing to import your NFTs in your WordPress media library, to include them in your website

### CODE REVIEW

File: projects/nft-import.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+## Target Audience <!--Describe who will be your project's users-->
+WordPress editors : photographs, artists, e-commerce, journalists, blogger...
+
+## Rough estimated user base (if applicable)
+WordPress is 40% of all web2 sites worldwide, possible target of hundred thousands editors 
+<!--How many users do you have right now?-->
+
+## Github repo <!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+https://github.com/Kredeum/ethglobal-web3
+
```
Comment (autonome) [inline projects/nft-import.md:50]: The grants program requires MIT license, if you are applying for that.

**Review by autonome — CHANGES_REQUESTED** [review]
Looks great, thanks for submitting! If you're applying for a Next Step grant however, the code must be MIT licensed.

File: projects/nft-import.md (hunk 1 above)
Comment (zapaz) [reply projects/nft-import.md:50]: ok, not so used to licence differences

repo set to MIT licence now

Comment (autonome) [reply projects/nft-import.md:52]: 👍🏼🙏🏼

**Pull request merged**